src/js/vre/chapter36/config/form.js

```diff
@@ -247,6 +288,227 @@ const formConfig = {
     additionalInformation: {
       title: 'Additional Information',
       pages: {
+        additionalInformationVeteran: {
+          title: 'Additional Information',
+          path: 'additional-information',
+          depends: {
+            'view:isVeteran': true
+          },
+          uiSchema: {
+            previousBenefitApplications: previousBenefitApplicationsUI
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              previousBenefitApplications: {
```

Comment:
This is defined differently in the schema. Should the configuration here (consistent with how we build checkbox fields elsewhere) be changed or the schema updated? The schema definition is:
```
    "previousBenefitApplications": {
      "type": "array",
      "items": {
        "type": "string"
      }
    },
```

Reply:
Maybe an array of [enum](https://stackoverflow.com/questions/30924271/correct-way-to-define-array-of-enums-in-json-schema)?

Reply:
I guess technically it would be oneOf either an enum of none or an enum of the other three options.

Reply:
I think we should update the schema. What you came up with is more consistent with how we've treated similar fields on other forms.